src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -1212,9 +1490,9 @@ public static partial class Kernel32
         /// </returns>
         [DllImport(api_ms_win_core_namedpipe_l1_2_0, SetLastError = true, CharSet = CharSet.Unicode)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static extern bool GetNamedPipeClientComputerName(
+        public static extern unsafe bool GetNamedPipeClientComputerName(
             SafeObjectHandle Pipe,
-            StringBuilder ClientComputerName,
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Array)] char* ClientComputerName,
```

Comment:
Another one of those changes that are consistent with our contribution guidelines, but changing an existing public method is an API breaking change. Can you add an overload instead of changing the existing one?